config/samples/backends/lvm-nvme-tcp/backend.yaml

```diff
@@ -0,0 +1,27 @@
+# This sample requires an LVM VG with the name of `cinder-volumes` to exist on the OpenShift node where cinder-volume runs
+# It also requires that nvme-fabrics kernel module is running everywhere and that nvmet is running on the controller host
+# The correct way to configure the load of additional kernel modules is using `MachineConfig` as shown in `nvme-fabrics.yaml`.yaml`.
+# To create the LVM on CRC the sample also uses a `MachineConfig` using a systemd unit with file `lvm.yaml`
+#
+# ATTENTION: After applying this OpenShift nodes will reboot, because of the `MachineConfig` changes and will take a while to recover.
+apiVersion: core.openstack.org/v1beta1
+kind: OpenStackControlPlane
+metadata:
+  name: openstack
+spec:
+  cinder:
+    template:
+      cinderVolumes:
+        lvm-nvme-tcp:
+          customServiceConfig: |
+            [lvm]
+            volume_backend_name=lvm_nvme_tcp
+            volume_driver=cinder.volume.drivers.lvm.LVMVolumeDriver
+            target_helper=nvmet
+            target_protocol=nvmet_tcp
+            target_port=4420
+            target_prefix=nvme-subsystem-1
+            nvmeof_conn_info_version=2
+            image_volume_cache_enabled=false
```

Comment:
I'd like to know if we should disable the image volume cache in every backend sample. My concern is users may infer this is deliberate and not something they would be comfortable overriding. In other words, are the sample suggesting they shouldn't enable the cache?

Reply:
For testing I always prefer to disable the cache because then during cleanup I don't have to remember to delete the cache volumes in the internal tenant project.
And I know it's the default, but I like things explicit.
Since this is a generic example for other people, and not just for me, I'll change it.

Reply:
Maybe we just need to document that these samples should not be taken as deployment "recommendations", just as working backend configurations and that they should look into the vendor's documentation for recommendations?